marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -147,7 +146,7 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
                             entityDefModulesFinder,
                             propertiesModuleManager,
                             entityResourceToURI,
-                            buildMetadataForEntityModels(hubConfig),
+                            buildMetadata(hubConfig, "http://marklogic.com/entity-services/models"),
```

Comment:
So for example, this would be buildMetadata(hubConfig.getEntityModelPermissions(), "the collection")